File: programmers/Level2/CPP/JadenCase 문자열 만들기.cpp
#include <string>
using namespace std;

string solution(string s) {
    string answer = s;
    
    bool b = 0;
    for(int i =0; i<answer.size(); i++){
        if(b == 0){
            b = 1;
            if(answer[i] >= 'a' && answer[i] <= 'z'){
                    answer[i] = answer[i] - 'a' + 'A';
            }
            if(answer[i] == ' ') {b = 0;}
        }
        else{
            if(answer[i] == ' ') {b = 0;}
            else { 
                if(answer[i] >= 'A' && answer[i] <= 'Z'){
                    answer[i] = answer[i] - 'A' + 'a';
                }
                b = 1;
            }
        }
    }
    
    return answer;
}

File: programmers/Level2/JAVA/JadenCase 문자열 만들기.java
class Solution {
    public String solution(String s) {
        String answer = "";
    
        boolean b = false;
        for(int i =0; i<s.length(); i++){
            if(b == false){
                b = true;
                if(s.charAt(i) == ' ') {answer+= ' '; b = false; continue;}
                else answer += Character.toUpperCase(s.charAt(i));
                b = true;
            }
            else{
                if(s.charAt(i) == ' ') {answer+= ' '; b = false; continue;}
                else answer += Character.toLowerCase(s.charAt(i));
                b = true;
            }
        }
    
        return answer;
    }
}
